roles/aodh/tasks/main.yml

```diff
@@ -18,7 +18,8 @@
     name: "{{ item.name }}"
     user: "{{ item.user }}"
     cmd: "{{ item.cmd }}"
-    config_dirs: "{{ item.config_dirs }}"
+    args: "{{ (item.name == 'openstack-aodh-api') | ternary('--port %s' % endpoints.aodh.port.backend_api, '') }}"
+    config_dirs: "{{ (item.name == 'openstack-aodh-api') | ternary('',{{ item.config_dirs }}) }}" 
```

Comment:
this will fail as u have {{ in ternary operation.
need something like
`config_dirs: "{{ (item.name == 'aodh-api') | ternary('', '%s' % item.config_dirs ) }}"`